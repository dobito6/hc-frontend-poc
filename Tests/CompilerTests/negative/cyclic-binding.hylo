fun main() {
  let y = y
}
