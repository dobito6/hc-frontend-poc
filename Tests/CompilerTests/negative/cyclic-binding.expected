negative/cyclic-binding.hylo:2.7-8: error: declaration refers to itself
  let y = y
      ^
